{{cookiecutter.project_name}}/docker/django/Dockerfile

```diff
@@ -42,7 +42,15 @@ RUN chmod +x "/docker-entrypoint.sh" \
   && poetry config settings.virtualenvs.create false \
   && poetry install $(test "$DJANGO_ENV" == production && echo "--no-dev") --no-interaction --no-ansi
 
-# Creating folders, and files for a project:
-COPY . /code
+# This dir will become the mountpoint of development code
+WORKDIR /code
 
 ENTRYPOINT ["/sbin/tini", "--", "/docker-entrypoint.sh"]
+
+
+FROM development_build as production_build
+
+COPY . /code
+
+# The presence of this file allows one to realize that a volume mounting has failed
+RUN touch "THESE_FILES_ARE_COPIES_OF_SOURCE_CODES"
```

Comment:
Can you please explain this to me? I have never seen something like this before.
Is it a hack or something?

Reply:
It's just a placeholder file created, so that user sees at first glance whether the VM he's in has copied file for production, or is still using a volume mount from sources. Since I don't know easy way to see this from inside the VM, I thought it'd be useful.

Reply:
I am not sure that this should be a part of the template. Let's move it to the docs as a life-hack 👍

Reply:
I removed it, people will easy add it to help debugging now that it's 2-stages